code/src/components/Form.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+
+import { UserName } from './Name'
+import { AgeRadioOption } from './Age'
+import { SelectPlanet } from './SelectPlanet'
+import { BringItem } from './BringItem'
+import { Summary } from './Summery'
+
+
+
+
+export const Form = () => {
+  const [ageGroupSpan, setAgeGroup] = useState('')
+  const [name, setName] = useState('')
+  const [item, setInputItem] = useState('')
+  const [planet, setPlanet] = useState('')
+  const [summary, setSummary] = useState(false)
+
+  const handelSubmit = event => {
+    event.preventDefault()
+    setSummary(true)
+  }
+
+ return (
+  <>
+   {! summary ? (
+    <form className='form-container' onSubmit={handelSubmit}>
+      <div className='instruction-text-container'>
+        <p>To secure a spot in the evacuation progam.</p>
+        <p>Please fill in the form.</p>
+      </div>
+      <div>
```

Comment:
Do you need to have a div around these mounted components in the form.js-file as well as in the respective component-files? 🧐 One suggestion is to remove the div’s from the form.js-file and only have them in the component-files.